pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -29,30 +29,32 @@ var (
 	AuthenticationStage StageName = "authentication"
 	ProductsStage       StageName = "products"
 
-	ProductAMQStreams          ProductName = "amqstreams"
-	ProductAMQOnline           ProductName = "amqonline"
-	ProductSolutionExplorer    ProductName = "solution-explorer"
-	ProductRHSSO               ProductName = "rhsso"
-	ProductRHSSOUser           ProductName = "rhssouser"
-	ProductCodeReadyWorkspaces ProductName = "codeready-workspaces"
-	ProductFuse                ProductName = "fuse"
-	ProductFuseOnOpenshift     ProductName = "fuse-on-openshift"
-	Product3Scale              ProductName = "3scale"
-	ProductNexus               ProductName = "nexus"
-	ProductLauncher            ProductName = "launcher"
-	ProductUps                 ProductName = "ups"
-	ProductMonitoring          ProductName = "monitoring"
+	ProductAMQStreams             ProductName = "amqstreams"
+	ProductAMQOnline              ProductName = "amqonline"
+	ProductSolutionExplorer       ProductName = "solution-explorer"
+	ProductRHSSO                  ProductName = "rhsso"
+	ProductRHSSOUser              ProductName = "rhssouser"
+	ProductCodeReadyWorkspaces    ProductName = "codeready-workspaces"
+	ProductFuse                   ProductName = "fuse"
+	ProductFuseOnOpenshift        ProductName = "fuse-on-openshift"
+	Product3Scale                 ProductName = "3scale"
+	ProductNexus                  ProductName = "nexus"
+	ProductLauncher               ProductName = "launcher"
+	ProductUps                    ProductName = "ups"
+	ProductMonitoring             ProductName = "monitoring"
+	ProductMobileDeveloperConsole ProductName = "mdc"
 
 	// PBrookes 08/08/2019:
 	// Could not find a way to determine these versions dynamically, so they are hard-coded
 	// It is preferable to determine the version of a product dynamically (from a CR, or configmap, etc)
-	VersionAMQOnline           ProductVersion = "1.2.2"
-	VersionAMQStreams          ProductVersion = "1.1.0"
-	VersionCodeReadyWorkspaces ProductVersion = "1.2.0.GA"
-	VersionFuseOnOpenshift     ProductVersion = "master"
-	VersionNexus               ProductVersion = "3.16"
-	VersionLauncher            ProductVersion = "0.1.2"
-	VersionUps                 ProductVersion = "2.3.2"
+	VersionAMQOnline              ProductVersion = "1.2.2"
+	VersionAMQStreams             ProductVersion = "1.1.0"
+	VersionCodeReadyWorkspaces    ProductVersion = "1.2.0.GA"
+	VersionFuseOnOpenshift        ProductVersion = "master"
+	VersionNexus                  ProductVersion = "3.16"
+	VersionLauncher               ProductVersion = "0.1.2"
+	VersionUps                    ProductVersion = "2.3.2"
+	VersionMobileDeveloperConsole ProductVersion = "1.1.11"
```

Comment:
this should only be added this way, if there is no way to find the version dynamically, is that the case?